tests/rgw/sanity_rgw_multisite.py

```diff
@@ -22,7 +22,7 @@ def run(**kw):
 
     # adding sleep for 60 seconds before another test starts, sync needs to complete
     time.sleep(60)
-    verify_io_on_sites = config.get("verify-io-on-sites", [])
+    #verify_io_on_sites = config.get("verify-io-on-sites", [])
```

Comment:
I think using the lists is better here, since we may choose to run verification on the same node if desired, and if that happens all we have to do is pass the cluster name in the yaml under `verify-io-on-sites`